internal/postgres/reconcile.go

```diff
@@ -293,9 +293,9 @@ func PodSecurityContext(cluster *v1beta1.PostgresCluster) *corev1.PodSecurityCon
 	// - https://cloud.redhat.com/blog/a-guide-to-openshift-and-uids
 	// - https://docs.k8s.io/tasks/configure-pod-container/security-context/
 	// - https://docs.openshift.com/container-platform/4.8/authentication/managing-security-context-constraints.html
-	if cluster.Spec.OpenShift == nil || !*cluster.Spec.OpenShift {
-		podSecurityContext.FSGroup = initialize.Int64(26)
+	if !initialize.FromPointer(cluster.Spec.OpenShift) {
```

Comment:
ah, OK, had to read the func / re-read the word "initialize" to get it: we give the value of the pointer or the null value of the type if the pointer is null.
 
So: Spec.OpenShift: true => true ; and false/nil => false.

Reply:
Yeah, this call doesn't read well. A function that returns zero when a pointer is nil _sounds_ intuitive, but I dunno in practice. Other functions in this package act on the pointers their passed, so that combo is confusing, too.

K8s has a not-versioned pile of `util` with a [ptr](https://pkg.go.dev/k8s.io/utils/ptr) `Deref(p *T, default T) T`. That doesn't read much better to me.